docs/content/connectors/vitess-cdc.md

```diff
@@ -0,0 +1,309 @@
+# Vitess CDC Connector
+
+The Vitess CDC connector allows for reading of incremental data from Vitess cluster. The connector does not support snapshot feature at the moment. This document describes how to setup the Vitess CDC connector to run SQL queries against Vitess databases.
+[Vitess debezium documentation](https://debezium.io/documentation/reference/connectors/vitess.html)
+
+Dependencies
+------------
+
+In order to setup the Vitess CDC connector, the following table provides dependency information for both projects using a build automation tool (such as Maven or SBT) and SQL Client with SQL JAR bundles.
+
+### Maven dependency
+
+```
+<dependency>
+  <groupId>com.ververica</groupId>
+  <artifactId>flink-connector-vitess-cdc</artifactId>
+  <version>2.0.0</version>
```

Comment:
The version should be 2.4-SNAPSHOT on master branch. Same issue for the download link below